### ISSUE TITLE
Add test to layout intrinsic asserting that natural size of image does not impact amp-img size

### ISSUE THREAD

**Issue by cathyxz** [ampproject/amphtml#24512]
Follow up to https://github.com/ampproject/amphtml/issues/24366. We should have a test explicitly defining the behavior of layout intrinsic and asserting that the size of the amp-img is governed by the height and width attributes passed to it, and not by the image's natural size. The test should pass height and width attributes to the image that are different from the image's natural size, and assert that the size of the amp-img is matches the former and not the latter.
